Makefile

```diff
@@ -454,12 +454,12 @@ prepare-bundle-to-quay:
 .PHONY: build-bundle-index-image
 ## build bundle index image
 build-bundle-index-image:
-	$(eval TMP_BUNDLE_IMAGE := localhost:5555/tmp-bundle:$(BUNDLE_VERSION))
+	$(eval BUNDLE_IMAGE := $(OPERATOR_BUNDLE_IMAGE):bundle-$(BUNDLE_VERSION))
 	$(Q)-$(CONTAINER_RUNTIME) rm -f tmp-registry
 	$(Q)$(CONTAINER_RUNTIME) run -d --name tmp-registry -p 5555:5000 registry:2
-	$(Q)operator-sdk bundle create --directory $(MANIFESTS_TMP)/$(BUNDLE_VERSION) -b $(CONTAINER_RUNTIME) --package $(CSV_PACKAGE_NAME) --channels beta,alpha --default-channel beta $(VERBOSE_FLAG) $(TMP_BUNDLE_IMAGE)
-	$(Q)$(CONTAINER_RUNTIME) push $(TMP_BUNDLE_IMAGE)
-	$(Q)opm index add -u $(CONTAINER_RUNTIME) -p $(CONTAINER_RUNTIME) --bundles $(TMP_BUNDLE_IMAGE) --tag "$(OPERATOR_BUNDLE_IMAGE):index"
+	$(Q)operator-sdk bundle create --directory $(MANIFESTS_TMP)/$(BUNDLE_VERSION) -b $(CONTAINER_RUNTIME) --package $(CSV_PACKAGE_NAME) --channels beta,alpha --default-channel beta $(VERBOSE_FLAG) $(BUNDLE_IMAGE)
+	$(Q)$(CONTAINER_RUNTIME) push $(BUNDLE_IMAGE)
```

Comment:
This is needed, because the OLM (probably) pulls the bundle image while installing it from CatalogSource. Otherwise it will fail, while complaining that it could not find the image. The following snippet shows the logs of a pod used by OLM to install the bundle (in case the `localhost:5555:/tmp-bundle:<bundle-version>` was used:
```
...
98s         Normal    Pulling             pod/4f7635c7bdfab8029f0170015dd41b2e263ad57e33197b38b0ae01cfeezbnjp   Pulling image "localhost:5555/tmp-bundle:0.3.0-446"
98s         Warning   Failed              pod/4f7635c7bdfab8029f0170015dd41b2e263ad57e33197b38b0ae01cfeezbnjp   Failed to pull image "localhost:5555/tmp-bundle:0.3.0-446": rpc error: code = Unknown desc = error pinging docker registry localhost:5555: Get "http://localhost:5555/v2/": dial tcp [::1]:5555: connect: connection refused
86s         Normal    BackOff             pod/4f7635c7bdfab8029f0170015dd41b2e263ad57e33197b38b0ae01cfeezbnjp   Back-off pulling image "localhost:5555/tmp-bundle:0.3.0-446"
...
```